doc/flatpak.xml

```diff
@@ -87,6 +87,19 @@
             the separate 
             <citerefentry><refentrytitle>flatpak-builder</refentrytitle><manvolnum>1</manvolnum></citerefentry> tool.
         </para>
+
+        <para>
+            Flatpak supports installing from sideload repos. These are partial
+            copies of a repository (generated by flatpak create-usb) that are used as
+            installation source when offline (and online as a performance improvement).
+            Such  repositories are configured by creating symlinks to the sideload sources
+            in the <filename>sideload-repos</filename>  subdirectory of the installation directory (i.e. typically
+            <filename>/var/lib/flatpak/sideload-repos</filename>). Additionally
+            symlinks can be created in <filename>/run/flatpak/sideload-repos</filename>
+            (overridable by <envar>FLATPAK_RUN_DIR</envar>) which is a better location
+            for non-persistent sources (as it is cleared on reboot).
```

Comment:
Would be nice to mention that by default the repo created by `create-usb` is at the path `.ostree/repo`.